layouts/docs/baseof.print.html

```diff
@@ -3,19 +3,15 @@
   <head>
     {{ partial "head.html" . }}
   </head>
-  <body class="td-{{ .Kind }}">
-<!-- MvM - deactivation of original navbar - as in baseof.html
+  <body class="td-{{ .Kind }}{{ with .Page.Params.body_class }} {{ . }}{{ end }}">
+{{/* MvM - deactivation of original navbar - as in baseof.html
     <header>
       {{ partial "navbar.html" . }}
     </header>
--->
+*/}}
     <div class="container-fluid td-outer">
       <div class="td-main">
         <div class="row flex-xl-nowrap stretch-main">
-          <div class="col-12 col-md-3 col-xl-2 td-sidebar d-print-none">
-          </div>
-          <div class="d-none d-xl-block col-xl-2 td-toc d-print-none">
-          </div>
           <main class="col-12 col-md-9 col-xl-8 pl-md-5" role="main">
```

Comment:
This displays the print version on the whole screen without empty columns on either side
```suggestion
          <main class="col-12 col-md-12 col-xl-12 pl-md-5" role="main">
```
Can you try this out and see if it looks OK for you.

Reply:
Fixed with `<main class="col-12 pl-md-5" role="main">`